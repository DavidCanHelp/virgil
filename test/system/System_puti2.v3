component System_puti2 {
	def main(arg: Array<string>) -> int {
		var f = System.puti;
		f(0);
		f(-1);
		f(1);
		f(200);
		f(-2147483648);
		f(2147483647);
		return 0;
	}
}
