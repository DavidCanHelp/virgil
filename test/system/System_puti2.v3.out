0-11200-21474836482147483647